### ISSUE TITLE
Couldn't find eliable source

### ISSUE THREAD

**Issue by rbz205** [DxCx/plugin.video.9anime#64]
I am getting above error. I just downloaded most current zip, but it is still an issue. 

Animes tried are Shingenki no kyojin season 2; Gabriel Dropout; Shingenki Kyojin Chuugakkou;

Could it be a site issue?

**Comment by hoppler** [comment]
@rbz205 Did you try 9anime in your browser on the pc and did it work for you?

**Comment by rbz205** [comment]
Yes, site works fine on a browser. 

Just tried it again in Kodi and no luck.

**Comment by Neogrey** [comment]
Currently only mycloud and openload are available, which should be working with the plugin.

> F1,F2,F4,AutoNext are not available for now, we will try to bring it back if possible, otherwise please don't complain or mail us about this issue. 

My error:
```
20:07:52.413 T:140217932764928   DEBUG: CAnnouncementManager - Announcement: OnClear from xbmc
20:07:52.414 T:140217932764928   DEBUG: GOT ANNOUNCEMENT, type: 2, from xbmc, message OnClear
20:07:52.415 T:140218467606208   DEBUG: StartScript - calling plugin 9Anime('plugin://plugin.video.9anime/play/alice-to-zouroku.532v/9','21','')
20:07:52.415 T:140217747506944   DEBUG: Thread LanguageInvoker start, auto delete: false
20:07:52.415 T:140217747506944    INFO: initializing python engine.
20:07:52.415 T:140217747506944   DEBUG: CPythonInvoker(23, /home/patrik/.kodi/addons/plugin.video.9anime/default.py): start processing
20:07:52.418 T:140217932764928   DEBUG: CAnnouncementManager - Announcement: OnAdd from xbmc
20:07:52.419 T:140217932764928   DEBUG: GOT ANNOUNCEMENT, type: 2, from xbmc, message OnAdd
20:07:52.423 T:140217747506944   DEBUG: -->Python Interpreter Initialized<--
20:07:52.423 T:140217747506944   DEBUG: CPythonInvoker(23, /home/patrik/.kodi/addons/plugin.video.9anime/default.py): the source file to load is "/home/patrik/.kodi/addons/plugin.video.9anime/default.py"
20:07:52.424 T:140217747506944   DEBUG: CPythonInvoker(23, /home/patrik/.kodi/addons/plugin.video.9anime/default.py): setting the Python path to /home/patrik/.kodi/addons/plugin.video.9anime:/home/patrik/.kodi/addons/script.common.plugin.cache/lib:/home/patrik/.kodi/addons/script.module.requests/lib:/home/patrik/.kodi/addons/script.module.urlresolver/lib:/usr/lib/python27.zip:/usr/lib64/python2.7:/usr/lib64/python2.7/plat-linux2:/usr/lib64/python2.7/lib-tk:/usr/lib64/python2.7/lib-old:/usr/lib64/python2.7/lib-dynload:/usr/lib64/python2.7/site-packages:/usr/lib64/python2.7/site-packages/gtk-2.0:/usr/lib/python2.7/site-packages
20:07:52.424 T:140217747506944   DEBUG: CPythonInvoker(23, /home/patrik/.kodi/addons/plugin.video.9anime/default.py): entering source directory /home/patrik/.kodi/addons/plugin.video.9anime
20:07:52.424 T:140217747506944   DEBUG: CPythonInvoker(23, /home/patrik/.kodi/addons/plugin.video.9anime/default.py): instantiating addon using automatically obtained id of "plugin.video.9anime" dependent on version 2.25.0 of the xbmc.python api
20:07:52.435 T:140217302890240   DEBUG: Thread scriptobs start, auto delete: false
20:07:52.635 T:140218467606208   DEBUG: ------ Window Init (DialogBusy.xml) ------
20:07:53.595 T:140217747506944   DEBUG: DialogProgress::Open called
20:07:53.595 T:140217747506944   DEBUG: ------ Window Init (DialogConfirm.xml) ------
20:07:53.601 T:140217747506944   DEBUG: Probing source: http://9anime.is/watch/alice-to-zouroku.532v/kx1pmr
20:07:54.864 T:140217747506944   DEBUG: [*E*]: error while trying to fetch info: token
20:07:54.864 T:140217747506944   DEBUG: [*E*] No extractor found for https://9anime.is/watch/alice-to-zouroku.532v/kx1pmr
20:07:54.864 T:140217747506944   DEBUG: Skipping invalid source MyCloud
20:07:54.865 T:140217747506944   DEBUG: Probing source: http://9anime.is/watch/alice-to-zouroku.532v/j219ly
20:07:55.709 T:140218467606208    INFO: CheckIdle - Closing session to http://2.bp.blogspot.com (easy=0x7f86f4024a40, multi=0x7f86f4048380)
20:07:55.709 T:140218467606208    INFO: CheckIdle - Closing session to http://2.bp.blogspot.com (easy=0x7f86a800c2c0, multi=(nil))
20:07:56.133 T:140217747506944   DEBUG: [*E*]: error while trying to fetch info: token
20:07:56.133 T:140217747506944   DEBUG: [*E*] No extractor found for https://9anime.is/watch/alice-to-zouroku.532v/j219ly
20:07:56.133 T:140217747506944   DEBUG: Skipping invalid source OpenLoad
20:07:56.143 T:140217747506944    INFO: CPythonInvoker(23, /home/patrik/.kodi/addons/plugin.video.9anime/default.py): script successfully run
20:07:56.154 T:140217302890240   DEBUG: Thread scriptobs 140217302890240 terminating
20:07:56.154 T:140218467606208   ERROR: Playlist Player: skipping unplayable item: 0, path [plugin://plugin.video.9anime/play/alice-to-zouroku.532v/9]
20:07:56.154 T:140218467606208   DEBUG: Playlist Player: one or more items failed to play... aborting playback
```

**Comment by p1wookie** [comment]
Just ran into this with Dragon Ball Super. Works fine in browser.

**Comment by hoppler** [comment]
@all seen it, seems like token changed

**Comment by DxCx** [comment]
hey, just seen it myself, they updated it overnight..
don't have much time now but i can see that info url now requires 2 more params
`?ts=1497164400&_=2487`
looks like, ts is easily obtainable from body (<body data-ts="1497164400">)
but we need to find where _ coming from.
im guessing from /token endpoint.
@githubus11 would you mind check that?

**Comment by grabberboy** [comment]
already seen the page and try to get the _=

its not on token?v1 url
its on all.js

someone can find the _= variable please on all.js
i cant find how they assign it on each info

**Comment by hoppler** [comment]
i have the ts variable: its $('body')['data']('ts')

**Comment by grabberboy** [comment]
the _ variable is the most important
hope we can find it

**Comment by grabberboy** [comment]
@hoppler can you share the code here so we can fix our plugin too?

Thanks

**Comment by hoppler** [comment]
@grabberboy @DxCx i made a pull for the changes

**Comment by grabberboy** [comment]
@hoppler 
your a life saver bro!
if ever i want to know your email
if ever your interested as a freelancer

Thank you bro

**Comment by DxCx** [comment]
just released version 0.0.9 with the fix, if you have repo installed you should get it automaticlly.
enjoy 🎉
